### ISSUE TITLE
It's not possible to remove a deleted package

### ISSUE THREAD

**Issue by ghost** [terramar-labs/packages#26]
I use packages 3.1.0 together with gitlab 8.0.2.
After deleting a repository from gitlab, the package is still present inside packages, also if I sync remotes.
There is no delete function inside packages and if I try to deactivate the package I get an error message:

```
Fatal error: Uncaught exception 'Gitlab\Exception\RuntimeException' with message '404 Project Not Found' in /var/www/vhosts/domain.tld/packages/vendor/m4tthumphrey/php-gitlab-api/lib/Gitlab/HttpClient/Listener/ErrorListener.php:60 Stack trace: #0 /var/www/vhosts/domain.tld/packages/vendor/m4tthumphrey/php-gitlab-api/lib/Gitlab/HttpClient/HttpClient.php(177): Gitlab\HttpClient\Listener\ErrorListener->postSend(Object(Gitlab\HttpClient\Message\Request), Object(Gitlab\HttpClient\Message\Response)) #1 /var/www/vhosts/domain.tld/packages/vendor/m4tthumphrey/php-gitlab-api/lib/Gitlab/HttpClient/HttpClient.php(109): Gitlab\HttpClient\HttpClient->request('projects/121', Array, 'GET', Array) #2 /var/www/vhosts/domain.tld/packages/vendor/m4tthumphrey/php-gitlab-api/lib/Gitlab/Api/AbstractApi.php(50): Gitlab\HttpClient\HttpClient->get('projects/121', Array, Array) #3 /var/www/ in /var/www/vhosts/domain.tld/packages/vendor/m4tthumphrey/php-gitlab-api/lib/Gitlab/HttpClient/Listener/ErrorListener.php on line 60
```

**Comment by mikspark** [comment]
:+1: same problem, we change project namespace in gitlab, i can disable old package without error, but can't get new package. manual removing from db resolve my problem.

**Comment by tyler-sommer** [comment]
This is fixed in 3.2.0
